docs/src/quickstart/install.rst

```diff
@@ -44,7 +44,11 @@ according to the system used:
 
 The simplest way of installing Cython is by using ``pip``::
 
-  pip install Cython
+  pip install Cython[setuptools]
```

Comment:
I was thinking that I would add only a note about setuptools dependency instead of putting it directly to example but I suppose this is what 99% of users will do. Also I intentionally not mentioned python3.12 (which is version of python when user needs to use setuptools), but I wanted to keep it simple and straightforward. Moreover it is better to start users teaching to use setuptools instead of distutils.